CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 - Update libraries for nFlow Explorer. Includes fix for morgan library security issue.
   - https://github.com/NitorCreations/nflow/network/alert/nflow-explorer/package-lock.json/morgan/open
 - Fix travis build to actually run unit tests for nflow-explorer module.
+- Add possibility to pause and resume WorkflowLifecycle.
```

Comment:
Maybe explain for nFlow users (non-nFlow-developers) what pausing WorkflowLifecycle means in practise?